### PR TITLE
[react-mdl] Fixed title type of HeaderRowProps in 'index.d.ts'.

### PR DESCRIPTION
**Pull request by benishouga**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://react-mdl.github.io/react-mdl/components/layout/
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

File: types/react-mdl/index.d.ts
Comment (sandersn) [inline types/react-mdl/index.d.ts]: I know the documentation says any, but if you know it's `string | JSX.Element`, why not set it as that? Or if it really can be `any`, then I would remove the comment.

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/react-mdl/index.d.ts
Comment (benishouga) [reply types/react-mdl/index.d.ts]: I fixed it according to the code `HeaderProps` and the [document](https://react-mdl.github.io/react-mdl/components/layout/).

> ```typescript
> interface HeaderProps extends __MDLOtherProps {
>     title?: any; // string | JSX.Element
>     scroll?: boolean;
>     seamed?: boolean;
>     transparent?: boolean;
>     waterfall?: boolean;
>     hideTop?: boolean;
>     hideSpacer?: boolean;
> }
> ```

> Element | Prop | Type | Effect | Remarks
> ------- | ---- | ---- | ------ | -------
> Header | title | Any | Set the layout title | Optional
> HeaderRow | title | Any | Set the layout title | Optional

But I think `ReactNode` may be better now. Is this correct ?
If it does not matter, I'll fix `HeaderProps` too.

Comment (benishouga) [reply types/react-mdl/index.d.ts]: Sorry. I try to fix it, `HeaderProps` and `HeaderRowProps` inherit `React.HTMLAttributes`, `title` conflicts.

```typescript
interface HTMLAttributes<T> extends DOMAttributes<T> {
    // ...
    title?: string;
    // ...
}
```

What should I do？

Comment (sandersn) [reply types/react-mdl/index.d.ts]: That makes sense. I think it is fine to use `any` in that case.

**Review by sandersn — APPROVED** [review]

**Pull request merged**
